route_unspecified.go

```diff
@@ -2,10 +2,38 @@
 
 package netlink
 
+import (
```

Comment:
In this file must go only unimplemented methods with pointer receivers specific to routing ex `*Route`, `*NexthoInfo`